### PR TITLE
Fix worker cache refactor

### PR DESCRIPTION
**Pull request by bnjjj**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/worker/internal/handler_cache.go [hunk 1]
```diff
@@ -4,16 +4,20 @@ import (
...
 	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/gorilla/mux"
+	"github.com/ovh/cds/engine/worker/pkg/workerruntime"
```
Comment (sguiheux) [inline engine/worker/internal/handler_cache.go]: need space

File: engine/worker/internal/handler_cache.go [hunk 2]
```diff
@@ -4,16 +4,20 @@ import (
...
 	"github.com/ovh/cds/sdk"
 	"github.com/ovh/cds/sdk/log"
+	"github.com/spf13/afero"
```
Comment (sguiheux) [inline engine/worker/internal/handler_cache.go]: wrong place

File: engine/worker/internal/action/builtin_serve_static_files.go [hunk 3]
```diff
@@ -10,6 +10,8 @@ import (
 
 	"github.com/ovh/cds/engine/worker/pkg/workerruntime"
 	"github.com/ovh/cds/sdk"
+
+	"github.com/spf13/afero"
```
Comment (sguiheux) [inline engine/worker/internal/action/builtin_serve_static_files.go]: wrong place

File: engine/worker/cmd.go [hunk 4]
```diff
@@ -161,6 +162,12 @@ func initFromFlags(cmd *cobra.Command, w *internal.CurrentWorker) {
 		os.Exit(4)
 	}
 
+	basedir, err := filepath.EvalSymlinks(basedir)
+	if err != nil {
+		log.Error(context.TODO(), "symlink error: %v", err)
```
Comment (yesnault) [inline engine/worker/cmd.go]:
```suggestion
		log.Error(context.Background(), "symlink error: %v", err)
```

**Pull request merged**